remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/ConfigCache.kt

```diff
@@ -84,6 +90,20 @@ internal class ConfigCache @VisibleForTesting constructor(
     } else {
         emptyMap()
     }
+
+    private fun fetchConfig() {
```

Comment:
Shouldn't this also be a `suspend` function? I think that the `ConfigFetcher#fetch` function also needs to be updated to a `suspend` function so we don't accidentally use it outside of a coroutine.

Reply:
changed to suspend